private-templates-service/azuredeploy.json

```diff
@@ -39,17 +28,18 @@
     }
   },
   "variables": {
-    "linuxFxVersion": "DOCKER|acmsregistry.azurecr.io/acmsimage",
+    "linuxFxVersion": "DOCKER|gracez72/acms:1.0",
```

Comment:
is adding gracez72 temporary?

Reply:
Yep, I'll ask Matt if there's a Microsoft or adaptive card dockerhub we can upload to